scenario/GovernanceScenario.ts

```diff
@@ -332,4 +332,54 @@ async function fastL1ToPolygonGovernanceExecute(
     ['sendMessageToChild(address,bytes)'],
     [sendMessageToChildCalldata]
   );
-}
\ No newline at end of file
+}
+
+scenario(
+  'execute Optimism governance proposal',
+  {
+    filter: async (ctx) => isBridgedDeployment(ctx) && matchesDeployment(ctx, [{network: 'optimism'}]),
+  },
+  async ({ timelock, bridgeReceiver }, _context, world) => {
+    const governanceDeploymentManager = world.auxiliaryDeploymentManager;
+    if (!governanceDeploymentManager) {
+      throw new Error('cannot execute governance without governance deployment manager');
+    }
+
+    const proposer = await impersonateAddress(governanceDeploymentManager, COMP_WHALES.mainnet[0]);
+
+    const currentTimelockDelay = await timelock.delay();
+    const newTimelockDelay = currentTimelockDelay.mul(2);
+
+    const l2ProposalData = utils.defaultAbiCoder.encode(
+      ['address[]', 'uint256[]', 'string[]', 'bytes[]'],
+      [
+        [timelock.address],
+        [0],
+        ['setDelay(uint256)'],
+        [utils.defaultAbiCoder.encode(['uint'], [newTimelockDelay])]
+      ]
+    );
+
+    const sendMessageCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'bytes', 'uint32'],
+      [bridgeReceiver.address, l2ProposalData, 1000000]
```

Comment:
We need to be careful about calculating the `gasLimit` param and make sure that it's sufficient for the L2 txn. Also, I wonder what happens if the `gasLimit` we set is too low. Logically, the L2 transaction would fail but I don't see it covered in the docs.